aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/AliyunOSSMockLocalStore.java

```diff
@@ -37,8 +37,8 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import org.apache.commons.codec.binary.Hex;
```

Comment:
We try not to rely on Apache Commons for anything to avoid a direct dependency, even in tests (though it looks like that was already violated). Is there an easy way to replace this? We should be able to replace `FileUtils` with a `readFully` method in `IOUtil`

Reply:
commons code it was on the classpath already and i just needed *any* hex.encode call. There's a couple more...let me find them and see which is preferable